### ISSUE TITLE
Warning: none is not a known color

### ISSUE THREAD

**Issue by danajanezic** [preston/railroady#106]
I keep getting this when I run rake diagram:<any>, which then means that the background color in the svg's is black or when I convert to a png transparent.  Is there a reason the background color isn't set to white?

**Comment by preston** [comment]
Yes -- it's a common use case to render the output over non-white backgrounds, such as on web pages or in other documents. I'm not getting this error.. could you provide details on how to reproduce?
